js/gdax.js

```diff
@@ -499,9 +500,13 @@ module.exports = class gdax extends Exchange {
         return this.parseOrder (response);
     }
 
-    async cancelOrder (id, symbol = undefined, params = {}) {
+    async cancelOrder (id = undefined, symbol = undefined, params = {}) {
```

Comment:
Unfortunately, the `id` can't be undefined in case of `cancelOrder`... Because it's a method for canceling a single order by id. Therefore it should not be used to implement canceling all orders (without specifying the `id`). Instead, there's a couple of other methods, `cancelOrders (ids..., params...)` and `cancelAllOrders (params ...)`. You can use the search in files for those two terms to find the implementations for reference. Can I ask you to redo this in a unified way?

Reply:
Got it